tests/scattering1d/test_timefrequency_scattering.py

```diff
@@ -59,6 +59,63 @@ def test_Q():
     assert "Q_fr must be an integer or 1-tuple" in ve.value.args[0]
 
 
+def test_check_runtime_args():
+    kwargs = dict(J=10, J_fr=3, shape=4096, Q=8)
```

Comment:
Is Q2 customizable here? Do we need a test as well?

Reply:
yes, Q2 is customizable. `TimeFrequencyScatteringBase.build()` calls `ScatteringBase1D.build()`, so that's where it's being tested.

Reply:
> yes, Q2 is customizable. `TimeFrequencyScatteringBase.build()` calls `ScatteringBase1D.build()`, so that's where it's being tested.

Indeed. Thank you!